File: Inventory Management/wwwroot/js/Item/itemmodel.js
/// <reference path="../knockout.js" />

var itemModel = function (item)
{
    var self = this;
    item = item || {};
    self.id = ko.observable(item.id || 0);
    self.name = ko.observable(item.name || '');
    self.unit = ko.observable(item.unit || '');
    self.category = ko.observable(item.category || '');
}

File: Inventory.Entities/ItemModel.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace Inventory_Management.Models
{
    public class ItemModel
    {
        public int Id { get; set; }
        public string Name { get; set; }

        public string Unit {  get; set; }

        public string Category {  get; set; }

       
    }
}
